src/components/AllMovies/AllMovies.js

```diff
@@ -34,11 +35,24 @@ class AllMovies extends Component {
     return mappedMovies
   }
 
+
+
+  submitSearch = inputData => {
```

Comment:
I'm really excited for this filter method that isn't case sensitive and searches from partial words--hopefully we can get around to having the page auto-render with updated results as the user types